sentry_sdk/tracing.py

```diff
@@ -323,6 +323,13 @@ def __init__(
 
             self.scope = self.scope or hub.scope
 
+        if same_process_as_parent is not None:
```

Comment:
@sentrivana The parameter has a default value set to `True` and is typed as `bool` (`same_process_as_parent=True,  # type: bool`); hence, it is not possible to provide `None` to the constructor without either a type check warning or a `DeprecationWarning`.

Since not all internal uses are migrated, starting with `PendingDeprecationWarning` would be nicer.

Reply:
resolved in #4290